server/views/pages/liteApprove.html

```diff
@@ -0,0 +1,213 @@
+{% extends "../partials/layout.html" %}
+{% from "../macros/breadCrumb.html" import breadCrumb %}
+{% from "error-summary/macro.njk" import govukErrorSummary %}
+{% from "warning-text/macro.njk" import govukWarningText %}
+{% from "select/macro.njk" import govukSelect %}
+{% from "input/macro.njk" import govukInput %}
+{% from "character-count/macro.njk" import govukCharacterCount %}
+{% from "textarea/macro.njk" import govukTextarea %}
+{% from "button/macro.njk" import govukButton %}
+
+{% set pageTitle = "Other categories" %}
+
+{% block beforeContent %}
+
+  {% set breadCrumbList = [ {href: '/' + data.details.bookingId, title: 'Manage categorisations'} ] %}
+  {{ breadCrumb(pageTitle, breadCrumbList) }}
+
+  {% include "../partials/offenderBanner.html" %}
+
+{% endblock %}
+
+{% block content %}
+
+{% if errors.length > 0 %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errors
+  }) }}
+{% endif %}
+
+<div class="govuk-!-width-two-thirds govuk-body-s">
+
+  <h1 class="govuk-heading-l govuk-!-padding-top-4" id="previousCategoryHeading">Other category approval</h1>
+  <p>Approve a category assessment, exactly as in P-Nomis.</p>
+  {% if not liteInProgress %}
+
+    {{ govukWarningText({
+      text: 'There is nothing to approve: this person does not have a pending categorisation.',
+      iconFallbackText: "Warning",
+      classes: "forms-warning-text warning-overrides",
+      attributes: {  'id': 'inProgressWarning' }
+    }) }}
+
+    <div>
+      <a href="/" id="todoLink1">Todo list</a>
+      <a href="/{{ bookingId }}" id="manageLink1">Manage categorisations</a>
+    </div>
+
+  {% elif sameUser %}
+
```

Comment:
Do we enforce this for the main approvals?

Reply:
no but i put it in here simply because old nomis does it - i suppose users would be used to this